pysidt/sidt.py

```diff
@@ -339,6 +339,23 @@ def read_nodes(file):
 
 
 class MultiEvalSubgraphIsomorphicDecisionTree(SubgraphIsomorphicDecisionTree):
+    """
+    Makes prediction for a molecule based on multiple evaluations.
+
+    Args:
+        `decomposition`: method to decompose a molecule into groups.
+        `root_group`: root group for the tree
+        `nodes`: dictionary of nodes for the tree
+        `n_splits`: number of splits to try for each node. Default is 1.
```

Comment:
n_splits is the minimum number of disconnected structures that should be in the group. For example in bimolecular families if you form a bond between the two groups doing H_Abstraction you only have one structure with both reactive sites and you're actually doing intra_H_migration instead of H_Abstraction. n_splits stops us from forming improper internal bonds.

Reply:
hmmm this keyword variable name is really confusing. We should change it. What about `n_disconnected_struc`? "Splits" is a term with associated meaning in decision tree, so I don't think we should use this word like this.

Reply:
From subgroup meeting: change to `n_strucs`.